README.md

```diff
@@ -1,9 +1,50 @@
-[![Build Status](https://travis-ci.org/TREP-LABS/realdrip-backend.svg?branch=master)](https://travis-ci.org/TREP-LABS/realdrip-backend)
-[![Coverage Status](https://coveralls.io/repos/github/TREP-LABS/realdrip-backend/badge.svg?branch=master)](https://coveralls.io/github/TREP-LABS/realdrip-backend?branch=master)
-## RealDrip Backend
-`A node-js server using express-js
-`
-#### The Setup integrated the following CI/CD tools
- - TravisCI
- - Coveralls.io
- - Jest
\ No newline at end of file
+# RealDrip Backend [![Build Status](https://travis-ci.org/TREP-LABS/realdrip-backend.svg?branch=master)](https://travis-ci.org/TREP-LABS/realdrip-backend)  [![Coverage Status](https://coveralls.io/repos/github/TREP-LABS/realdrip-backend/badge.svg?branch=master)](https://coveralls.io/github/TREP-LABS/realdrip-backend?branch=master)
+
+> What is RealDrip? Check [here](https://treplabs.co/realdrip)
+
+This is the backend API of the RealDrip platform.
+
+## API Documentation
+The API is well documented using the OpenAPI Spec(Swagger) [here](https://app.swaggerhub.com/apis/Treplabs/Realdrip_Platform/1.0#/).
```

Comment:
There should be a space before the round bracket.
```suggestion
The API is well documented using the OpenAPI Spec (Swagger) [here]
```

Reply:
Why remove the link to the spec?